core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.html

```diff
@@ -1,67 +1,184 @@
+        </ul>
+      </div>
+      <div class="filter-container">
+        <div class="language-selector-container" *ngIf="(activeTab === TAB_NAME_TRANSLATION_SUBMITTER) || (activeTab === TAB_NAME_TRANSLATION_REVIEWER)">
+          <div #languageDropdown class="oppia-filter-selector-container e2e-test-language-selector"
+               (click)="toggleLanguageDropdown()">
+            <div class="oppia-filter-selector-inner-container e2e-test-language-selector-selected">
+              <b>Language: </b>{{selectedLanguage}}
+              <span class="fas fa-caret-down oppia-filter-selector-inner-container-arrow"
+                    [@languageDropdown]="languageDropdownShown ? 'expanded' : 'collapsed'">
+              </span>
+            </div>
+            <div *ngIf="languageDropdownShown"
+                 class="oppia-filter-selector-dropdown-container e2e-test-language-selector-dropdown">
+              <div class="oppia-filter-selector-dropdown-option e2e-test-language-selector-option"
+                   *ngFor="let option of languages"
+                   (click)="selectLanguage(option.language)"
+                   [ngClass]="{'oppia-filter-selector-dropdown-option-selected' : option === selectedLanguage}">
+                {{option.language}}
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="filter-sub-container">
+          <div class="last-activity-selector-container">
+            <div #activityDropdown  class="oppia-filter-selector-container e2e-test-last-activity-selector"
+                 (click)="toggleActivityDropdown()">
+              <div class="oppia-filter-selector-inner-container e2e-test-last-activity-selector-selected">
+                <b>Filter by last activity: </b>
+                   {{ selectedLastActivity ? 'Within past ' + selectedLastActivity + ' days' : 'All time' }}
+                <span class="fas fa-caret-down oppia-filter-selector-inner-container-arrow"
+                      [@lastActivityDropdown]="activityDropdownShown ? 'expanded' : 'collapsed'">
+                </span>
+              </div>
+              <div *ngIf="activityDropdownShown"
+                   class="oppia-filter-selector-dropdown-container e2e-test-last-activity-selector-dropdown">
+                <div class="oppia-filter-selector-dropdown-option e2e-test-last-activity-selector-option"
+                     *ngFor="let option of lastActivty"
+                     (click)="selectLastActivity(option)"
+                     [ngClass]="{'oppia-filter-selector-dropdown-option-selected' : option === selectedLastActivity}">
+                  {{ option ? 'Within past ' + option + ' days' : 'All time' }}
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="topic-selector-container">
+            <oppia-topic-filter
+              *ngIf="(activeTab === TAB_NAME_TRANSLATION_SUBMITTER) || (activeTab === TAB_NAME_QUESTION_SUBMITTER)"
```

Comment:
Why don't we have the topic filter for the translation reviewer tab?

Reply:
Actually all the filters have been planned and implemented in the backend itself. According to the [GSoC Proposal](https://docs.google.com/document/d/19lCEYQUgV7_DwIK_0rz3zslRHX2qKOHn-t9Twpi0qu0/edit#heading=h.vxqaj7i9qj8a), [Admin Dashboard PRD](https://docs.google.com/document/d/1_vvewEwviQjtPvZy9yo74eCcRTQNcQKyCgdHNpyWNWw/edit#heading=h.liiqbmwmz8j8) and [Mocks](https://www.figma.com/file/bmyOkQwosePcoNAr45Rdk0/Admin-Dashboard?node-id=0%3A1&mode=dev).

So we don't have topic filter for reviewers. FYI there is no topic filter for question reviewers as well.